<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="14" hostname="DESKTOP-L3M9CL3" failures="2" tests="4" name=" UsermasterPage" time="6.535" errors="0" timestamp="2022-10-20T11:58:27 IST">
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_001_VerifyWithValidCredentials" time="0.564"/>
  <testcase classname="com.Redipae.Testcases.AddUploadUser" name="TC_011_Validate_DuplicateUser" time="1.925">
    <failure type="java.lang.UnsupportedClassVersionError" message="com/microsoft/sqlserver/jdbc/SQLServerDriver has been compiled by a more recent version of the Java Runtime (class file version 62.0), this version of the Java Runtime only recognizes class file versions up to 61.0">
      <![CDATA[java.lang.UnsupportedClassVersionError: com/microsoft/sqlserver/jdbc/SQLServerDriver has been compiled by a more recent version of the Java Runtime (class file version 62.0), this version of the Java Runtime only recognizes class file versions up to 61.0
at java.base/java.lang.ClassLoader.defineClass1(Native Method)
at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1012)
at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:862)
at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
at java.base/java.lang.Class.forName0(Native Method)
at java.base/java.lang.Class.forName(Class.java:375)
at com.Redipae.Utilities.DBConfig.ConnectDB(DBConfig.java:14)
at com.Redipae.Testcases.AddUploadUser.TC_011_Validate_DuplicateUser(AddUploadUser.java:82)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_011_Validate_DuplicateUser -->
  <testcase classname="com.Redipae.Testcases.AddUploadUser" name="TC_010_Verify_AddUser" time="1.549">
    <failure type="java.lang.UnsupportedClassVersionError" message="com/microsoft/sqlserver/jdbc/SQLServerDriver has been compiled by a more recent version of the Java Runtime (class file version 62.0), this version of the Java Runtime only recognizes class file versions up to 61.0">
      <![CDATA[java.lang.UnsupportedClassVersionError: com/microsoft/sqlserver/jdbc/SQLServerDriver has been compiled by a more recent version of the Java Runtime (class file version 62.0), this version of the Java Runtime only recognizes class file versions up to 61.0
at java.base/java.lang.ClassLoader.defineClass1(Native Method)
at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1012)
at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:862)
at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
at java.base/java.lang.Class.forName0(Native Method)
at java.base/java.lang.Class.forName(Class.java:375)
at com.Redipae.Utilities.DBConfig.ConnectDB(DBConfig.java:14)
at com.Redipae.Testcases.AddUploadUser.TC_010_Verify_AddUser(AddUploadUser.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_010_Verify_AddUser -->
  <testcase classname="com.Redipae.Testcases.AddUploadUser" name="TC_012_Verify_TextboxErrormessage" time="2.472"/>
  <testcase classname="com.Redipae.Testcases.Setup" name="@BeforeSuite Login" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite Login -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_007_VerifyWithNull_Value_in_Password" time="0.0">
    <ignored/>
  </testcase> <!-- TC_007_VerifyWithNull_Value_in_Password -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_004_Verifywith_INValidCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- TC_004_Verifywith_INValidCredentials -->
  <testcase classname="com.Redipae.Testcases.AddUploadUser" name="TC_014_Verify_Export_USermaster" time="0.0">
    <ignored/>
  </testcase> <!-- TC_014_Verify_Export_USermaster -->
  <testcase classname="com.Redipae.Testcases.Setup" name="@AfterSuite closebrowser" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite closebrowser -->
  <testcase classname="com.Redipae.Testcases.AddUploadUser" name="TC_013_Verify_Upload_USermaster" time="0.0">
    <ignored/>
  </testcase> <!-- TC_013_Verify_Upload_USermaster -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_006_VerifyWithNull_Value_in_Username" time="0.0">
    <ignored/>
  </testcase> <!-- TC_006_VerifyWithNull_Value_in_Username -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_008_Verifywith_NumericValues" time="0.0">
    <ignored/>
  </testcase> <!-- TC_008_Verifywith_NumericValues -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="clickLogout" time="0.0">
    <ignored/>
  </testcase> <!-- clickLogout -->
  <testcase classname="com.Redipae.Testcases.AddUploadUser" name="TC_015_Verify_Exportwithdata" time="0.0">
    <ignored/>
  </testcase> <!-- TC_015_Verify_Exportwithdata -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_002_Verifywith_INValidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- TC_002_Verifywith_INValidPassword -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_005_VerifyWithNull_Value" time="0.0">
    <ignored/>
  </testcase> <!-- TC_005_VerifyWithNull_Value -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_009_Verifywith_SpecialCharacters" time="0.0">
    <ignored/>
  </testcase> <!-- TC_009_Verifywith_SpecialCharacters -->
  <testcase classname="com.Redipae.Testcases.LoginTest" name="TC_003_Verifywith_INValidUsername" time="0.0">
    <ignored/>
  </testcase> <!-- TC_003_Verifywith_INValidUsername -->
</testsuite> <!--  UsermasterPage -->
